erpnext/hr/report/monthly_attendance_sheet/monthly_attendance_sheet.js

```diff
@@ -35,15 +35,24 @@ frappe.query_reports["Monthly Attendance Sheet"] = {
 			"fieldname":"employee",
 			"label": __("Employee"),
 			"fieldtype": "Link",
-			"options": "Employee"
+			"options": "Employee",
+			get_query: () => {
+				var company = frappe.query_report.get_filter_value('company');
+				return {
+					filters: {
+						'company': company
+					}
+				};
+			}
 		},
 		{
 			"fieldname":"company",
 			"label": __("Company"),
 			"fieldtype": "Link",
 			"options": "Company",
 			"default": frappe.defaults.get_user_default("Company"),
-			"reqd": 1
+			"reqd": 1,
+
```

Comment:
```suggestion
			"reqd": 1
```
Remove the extra line and comma